File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer,Integer> mp=new HashMap<>();
        List<Integer> ls=new ArrayList<>();
        for(int num:nums){
            mp.put(num,mp.getOrDefault(num,0)+1);
        }

        List<Integer> keys = new ArrayList<>(mp.keySet());
        keys.sort((a, b) -> mp.get(b) - mp.get(a));

        for(int i = 0; i < k; i++) {
            ls.add(keys.get(i));
        }

        int ans[]=new int[ls.size()];

        for(int i=0;i<ls.size();i++){
            ans[i]=ls.get(i);
        }
        return ans;
    }
}

File: Data Structures & Algorithms/three-integer-sum/submission-1.py
class Solution:
    def threeSum(self, nums: List[int]) -> List[List[int]]:
        if not nums:
            return [[]]
        res=[]
        for i in range(0,len(nums)-2,1):
            for j in range(i+1,len(nums)-1,1):
                for k in range(len(nums)-1,j,-1):
                    summ=nums[i]+nums[j]+nums[k]
                    if summ==0:
                        ls=[nums[i],nums[j],nums[k]]
                        ls.sort()
                        if ls not in res:
                            res.append(ls)

        return res
        